nfd-daemonset-combined.yaml.template

```diff
@@ -104,6 +104,9 @@ spec:
             - name: host-sys
               mountPath: "/host-sys"
               readOnly: true
+            - name: host-usr
```

Comment:
Greetings !

can you update : https://github.com/kubernetes-sigs/node-feature-discovery/blob/09cb512f22bae327ac38699f63312728c39ceb38/deployment/node-feature-discovery/templates/worker.yaml#L60  as well ?

Reply:
Done. Thanks for the catch